src/map/ai/states/mobskill_state.cpp

```diff
@@ -102,6 +103,11 @@ bool CMobSkillState::Update(time_point tick)
     }
     if (IsCompleted() && tick > m_finishTime)
     {
+        auto PTarget = GetTarget();
+        if (PTarget && PTarget->objtype == TYPE_MOB)
+        {
+            static_cast<CMobEntity*>(PTarget)->PEnmityContainer->UpdateEnmity(m_PEntity, 0, 0, true);
```

Comment:
Will we run into any surprises if mobs of the same allegiance use buffing mobskills on each other (or a goblin drops his bomb near his buds)?